packages/sources/src/tests/__mocks__/FieldProvider.js

```diff
@@ -0,0 +1,24 @@
+import { createElement } from 'react';
```

Comment:
I am wondering if we should expose some testUtils in data driven forms.

Reply:
I was thinking about some test framework

Reply:
stuff like FieldProvider, some simple layout mapper etc. Let me think about it. We could also use it for testing our mappers etc.